gn3/api/heatmaps.py

```diff
@@ -0,0 +1,37 @@
+"""
+Module to hold the entrypoint functions that generate heatmaps
+"""
+
+import io
+from flask import jsonify
+from flask import request
+from flask import Blueprint
+from gn3.heatmaps import build_heatmap
+from gn3.db_utils import database_connector
+
+heatmaps = Blueprint("heatmaps", __name__)
+
+@heatmaps.route("/clustered", methods=("POST",))
+def clustered_heatmaps():
+    """
+    Parses the incoming data and responds with the JSON-serialized plotly figure
+    representing the clustered heatmap.
+    """
+    heatmap_request = request.get_json()
+    traits_names = heatmap_request.get("traits_names", tuple())
```

Comment:
*nitpick:* We only use the variable `heatmap_request` only once to get "trait_names". With naming being a chore, I would defer it here as such:
```suggestion
    traits_names = request.get_json().get("traits_names", tuple())
```

Opt out of creating variables that are only referenced once because it removes the extra cognitive load of coming up with names.

Reply:
@BonfaceKilz Fixed: commit a9fc9814760d205674904f8feb700eadae480fb1